tests/integration/targets/sysrc/aliases

```diff
@@ -0,0 +1,5 @@
+unsupported
```

Comment:
It is not possible to run tests in CI on FreeBSD?

Reply:
+1 to the question

Reply:
I copied this over from the original PR I created for ansible/ansible earlier this year. I'm not sure if FreeBSD was supported on that CI pipeline at the time.

I've removed the this and we can go from there.